### DISCUSSION TITLE
Do you have a plan to export the error codes of Prisma?

### DISCUSSION THREAD

**Discussion by ryochike** [prisma/prisma#18317] [Q&A] [answered]
Hi!
There is an Issue titled "[Export error code constants in npm package](https://github.com/prisma/prisma/issues/10394)" which raises the question why the error codes of Prisma are not exported as constants?

Currently, I'm mapping Prisma's error codes and creating error handling processes. However, I believe it would be difficult to handle changes to the error codes if they were to occur on the library side. Is there a plan to export them as constants? If not, I'm considering creating them myself.

**Comment by nurul3101** [comment, marked as answer]
Hi @Ryochike 👋 

Thanks for linking the feature request to export error codes, I believe the error codes would not be abruptly changed, but I can see how exporting error codes could be helpful. I'll discuss this internally with the team to get their insights on this.

**Reply by janpio** [reply]
Error codes are part of our public API, so if one would go away or would change it would be a breaking change and would only happen with a major version.
